node/impl/client/client_test.go

```diff
@@ -1,3 +1,4 @@
+//stm: #unit
```

Comment:
Can we deduce unit/integration based on directory?

Reply:
I think it's possible to do so (ie. tests in the `itests` folder are integration tests), though I'm not sure if anything more than that is too fragile and inflexible. 
I would suggest something different: the crawler can add an annotation `stm: #TODO` to the top of each test file that does not already have a specified annotation. These TODO's can then be shown as warnings in the dashboard, or in the CI, etc. This way, the developer is forced to fill in the correct type.